docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -21,6 +21,61 @@ Having defined a partitioned job or asset, you can:
 
 ---
 
+## Partitioned Assets
+
+### Relevant APIs
+
+| Name                                              | Description                                                                       |
+| ------------------------------------------------- | --------------------------------------------------------------------------------- |
+| <PyObject object="PartitionsDefinition" />        | Superclass - defines the set of partitions that can be materialized for an asset. |
+| <PyObject object="HourlyPartitionsDefinition" />  | A partitions definition with a partition for each hour.                           |
+| <PyObject object="DailyPartitionsDefinition" />   | A partitions definition with a partition for each day.                            |
+| <PyObject object="WeeklyPartitionsDefinition" />  | A partitions definition with a partition for each week.                           |
+| <PyObject object="MonthlyPartitionsDefinition" /> | A partitions definition with a partition for each month.                          |
+| <PyObject object="StaticPartitionsDefinition" />  | A partitions definition with a fixed set of partitions.                           |
+
+A software-defined asset can be assigned a <PyObject object="PartitionsDefinition" />, which determines the set of partitions that compose it. Once an asset has a set of partitions, you can launch materializations of individual partitions, as well as view the materialization history by partition in Dagit.
+
+Here's an asset with a partition for each day since the first day of 2022:
```

Comment:
```suggestion
For example, below is an asset with a partition for each day since the first day of 2022:
```

I feel like introducing examples lends itself to a better overall flow when reading, but this could just be my personal preference. Using words like 'below' to indicate where something is (like an example, in this case) can also be helpful for folks using screen readers.